### ISSUE TITLE
Contradiction in the documentation for the -UseMultithreading parameter of Connect-ExchangeOnline cmdlet

### ISSUE THREAD

**Issue by bajce** [MicrosoftDocs/office-docs-powershell#10757]
The [documentation page](https://learn.microsoft.com/en-us/powershell/module/exchange/connect-exchangeonline?view=exchange-ps#-usemultithreading) for the `Connect-ExchangeOnline cmdlet has a contradictory information regarding the -UseMultithreading parameter. Specifically, the description of the value $true states that it is the default value for this parameter. However, the "Default Value" section below contradicts this by stating that there is no default value.

This discrepancy could be confusing for users who are trying to understand the default behavior of this parameter.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 13d88f18-3cc8-5f18-8d61-ff7f5fa4e314
* Version Independent ID: 60f9a029-69f3-e65c-6816-fbd414790ce3
* Content: [Connect-ExchangeOnline (ExchangePowerShell)](https://learn.microsoft.com/en-us/powershell/module/exchange/connect-exchangeonline?view=exchange-ps#-usemultithreading)
* Content Source: [exchange/exchange-ps/exchange/Connect-ExchangeOnline.md](https://github.com/MicrosoftDocs/office-docs-powershell/blob/main/exchange/exchange-ps/exchange/Connect-ExchangeOnline.md)
* Service: **exchange-powershell**
* GitHub Login: @chrisda
* Microsoft Alias: **chrisda**

**Comment by chrisda** [comment]
@bajce, the "Default value" property on parameters has been a problem in any PowerShell content for years and years (possibly since the inception of PowerShell). The issue is: no tool or utility is able to programmatically capture the value of the Default value" property (unlike the "Type" or "Required" values, for example). Of course, many parameters don't have a default value, but many do (especially Boolean parameters).

Anywhere you see the "Default value" property populated in cmdlet reference content means someone has manually added a value to content. Prior to 2018 when Exchange-related cmdlet reference topics were on TechNet, we never even exposed that property in content (although it might have been visible from `Get-Help <Cmdlet>` at the command line.

In short, let the parameter description text tell you the default value of a parameter in Exchange cmdlet content.

**Comment by yogkumgit** [comment]
@bajce Hope this comment is helpful for you. If you see a documentation update is required, please feel free to open an issue for the same. We proceed here to close it. Thanks for taking out some time to open the issue. Appreciate and encourage you to do the same in future also.
